src/reactviews/pages/TableDesigner/designerPropertiesPane.tsx

```diff
@@ -92,108 +114,176 @@ export const DesignerPropertiesPane = () => {
                         );
-                    })}
+                    }}
+                    title={
+                        state.propertiesPaneResizeInfo.isMaximized
+                            ? locConstants.tableDesigner.restorePanelSize
+                            : locConstants.tableDesigner.maximizePanelSize
+                    }
+                    icon={
+                        state.propertiesPaneResizeInfo.isMaximized ? (
+                            <ChevronRightFilled />
+                        ) : (
+                            <ChevronLeftFilled />
+                        )
+                    }
+                />
+                <Text
+                    size={500}
+                    style={{
+                        fontWeight: "bold",
+                        flex: 1,
+                    }}
+                >
+                    {locConstants.tableDesigner.propertiesPaneTitle(
+                        parentTableProperties.objectTypeDisplayName ?? "",
+                    )}
+                </Text>
+                <Button
+                    appearance="outline"
+                    onClick={() => {
+                        state.provider.setPropertiesComponents(undefined);
+                    }}
+                    title={
+                        state.propertiesPaneResizeInfo.isMaximized
+                            ? locConstants.tableDesigner.restorePanelSize
+                            : locConstants.tableDesigner.maximizePanelSize
+                    }
+                    icon={<DismissRegular />}
+                />
+            </div>
+            <div className={classes.stack}>
+                <Accordion multiple collapsible defaultOpenItems={groups}>
+                    {data &&
+                        groups?.map((group) => {
+                            return (
+                                <AccordionItem value={group} key={group}>
+                                    <AccordionHeader>{group}</AccordionHeader>
+                                    <AccordionPanel>
+                                        <div className={classes.group}>
+                                            {parentTableProperties.itemProperties
+                                                ?.filter(
+                                                    (i) =>
+                                                        (group === "General" &&
+                                                            !i.group) ||
+                                                        group === i.group,
+                                                )
+                                                .filter(
+                                                    (item) =>
+                                                        item.showInPropertiesView !==
+                                                        false,
+                                                )
```

Comment:
When the code is indented this far, it gets annoying to read.  Some ideas to clean it up:

* Pull something (Accordion or AccordionPanel, maybe) out into a helper function/component.
* increase the prettier pagewidth
* pull the `.map()` contents out into a helper function